tests/integration/models/gpflux/test_predictions.py

```diff
@@ -12,84 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any
-
 import numpy as np
 import pytest
-import tensorflow as tf
-from gpflux.architectures import Config, build_constant_input_dim_deep_gp
 
 from tests.util.misc import hartmann_6_dataset, random_seed
-from tests.util.models.gpflux.models import trieste_deep_gaussian_process
-
-
-@pytest.fixture(name="depth", params=[2, 3])
-def _depth_fixture(request: Any) -> int:
-    return request.param
+from trieste.models.gpflux import DeepGaussianProcess, build_vanilla_deep_gp
+from trieste.objectives import HARTMANN_6_SEARCH_SPACE
 
 
 @pytest.mark.slow
 @random_seed
+@pytest.mark.parametrize("depth", [2, 3])
 def test_dgp_model_close_to_actuals(depth: int, keras_float: None) -> None:
     dataset_size = 50
     num_inducing = 50
-    batch_size = 50
-    epochs = 500
 
     example_data = hartmann_6_dataset(dataset_size)
 
-    model, _ = trieste_deep_gaussian_process(
-        query_points=example_data.query_points,
-        depth=depth,
-        num_inducing=num_inducing,
-        learning_rate=0.01,
-        batch_size=batch_size,
-        epochs=epochs,
-        fix_noise=True,
+    dgp = build_vanilla_deep_gp(
+        example_data,
+        HARTMANN_6_SEARCH_SPACE,
+        depth,
+        num_inducing,
+        likelihood_variance=1e-5,
+        trainable_likelihood=False,
     )
-
+    model = DeepGaussianProcess(dgp)
     model.optimize(example_data)
     predicted_means, _ = model.predict(example_data.query_points)
 
     np.testing.assert_allclose(predicted_means, example_data.observations, atol=0.2, rtol=0.2)
-
-
-@pytest.mark.slow
-@random_seed
-def test_dgp_model_close_to_simple_implementation(depth: int, keras_float: None) -> None:
```

Comment:
Why was this one deleted?

Reply:
in test_builders we already test that the structure is the same, so I thought this one is an overkill